HW1/P2/P2b.py

```diff
@@ -1,3 +1,21 @@
 from P2 import *
+import findspark
+findspark.init()
+import os
+import pyspark
+sc = pyspark.SparkContext()
 
-# Your code here
+# define shuffled axis: to ensure randomness, the axes are defined separately
+axis1 = range(2000)
+np.random.shuffle(axis1)
+axis2 = range(2000)
+np.random.shuffle(axis2)
```

Comment:
This is a good point. You can also cut down your number of lines of code here to make it a bit cleaner.